hugo/layouts/papers/list-entry.html

```diff
@@ -3,16 +3,13 @@
 <p class="d-sm-flex align-items-stretch">
   <span class="d-block mr-2 text-nowrap list-button-row">
     {{- with $paper.url -}}
-    <a class="badge badge-primary align-middle mr-1" href="{{ . }}" data-toggle="tooltip" data-placement="top" title="Open PDF">
+    <a class="badge badge-primary align-middle mr-1" href="{{ . }}.pdf" data-toggle="tooltip" data-placement="top" title="Open PDF">
       pdf
     </a>
-      {{- if and (hasPrefix . $.Site.Params.baseURL) (eq . (strings.TrimSuffix ".pdf" .)) -}}
-    <a class="d-none" href="{{ . }}.pdf" title="Hidden link to PDF with extension">pdf</a>
-      {{- end -}}
     {{- end -}}
```

Comment:
We had a hidden link to the PDF to help with Google, since the canonical URL had no extension. This is no longer needed.